repair.py

```diff
@@ -67,6 +68,29 @@ def main():
             getItems = lambda media, childId: arr.getFiles(media=media, childId=childId) if args.mode == 'symlink' else arr.getHistory(media=media, childId=childId, includeGrandchildDetails=True)
             childrenIds = media.childrenIds if args.include_unmonitored else media.monitoredChildrenIds
 
+            # perform a "Safety check" to make sure that the torrents folder is mounted to prevent deleting everything in *arrs
+            # we do this inside of the loop because the mount could drop at any time; so, best to check with each iteration

```

Comment:
Maybe only validate inside the loop(s) if either of the intervals have been entered.